src/content.js

```diff
@@ -471,6 +471,24 @@ function addTitleToEmojis() {
 	}
 }
 
+function shortenVisibleUrls() {
+	$('a[href]').get().forEach(a => {
```

Comment:
Prefer a `for-of` loop whenever possible as it's faster and more readable. Just use `continue` instead of `return`.